core/connection/manager.go

```diff
@@ -61,12 +64,21 @@ type Publisher interface {
 	Publish(topic string, args ...interface{})
 }
 
+// PaymentManager handles the payments for service
+type PaymentManager interface {
```

Comment:
Imho too abstract naming. It would be good, that interface naming we set could be easily relatable with original structure they represent. Without this relation is very hard to read code. Imho original naming with 'PaymentIssuer' or 'PromiseIssuer' would match better. Would be best if I search for 'NamedInterfaceAtUserSide' and I am able to find original struct where this interface is implemented. Not to traverse 100 dependency injections and look for it.

Reply:
renamed to payment issuer